just-download/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: just-download_web_1
+      APP_PORT: 5000
+  
+  web:
+    image: ghcr.io/highghlow/just-download:v1.0.6@sha256:9f7dadd49339acc09b84b70d67701ea77f6f3f32521ae69cc0f6845da9c9f07a
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      - LOCK_TRANSMISSION_CONFIG=1
+    volumes:
+      - ${APP_DATA_DIR}:/config
```

Comment:
This should be one level deeper at:
`${APP_DATA_DIR}/data:/config`

If you want to future proof it a bit you could do this instead:
`${APP_DATA_DIR}/data/config:/config`
so that if you go to add other bind mounts later on you can won't have to do any migrations (e.g. `${APP_DATA_DIR}/data/cache:/cache`)